lib/elixir/lib/kernel.ex

```diff
@@ -2948,11 +2948,18 @@ defmodule Kernel do
 
       Enum.member?([1, 2, 3], x)
 
+  Elixir also supports a `left not in right` syntax, which evaluates to
```

Comment:
Yeah leaving out the `a`/`the` entirely seems to make the most grammatical sense imho?